services/web/client/source/class/qxapp/component/form/renderer/PropForm.js

```diff
@@ -237,15 +237,15 @@ qx.Class.define("qxapp.component.form.renderer.PropForm", {
       return this._getChildren().filter(child => child.nodeId && this.__arePortsCompatible(output.nodeId, output.portId, child.nodeId, child.portId));
     },
 
-    __hightlightCompatibles: function(output) {
+    __highlightCompatibles: function(output) {
       const inputs = this.__getCompatibleInputs(output);
       for (let i in inputs) {
         const input = inputs[i];
         input.setDecorator("material-textfield-focused");
```

Comment:
the official way todo this would be to add a state and use this to trigger the correct theme rendering